src/core/evaluator.js

```diff
@@ -1324,6 +1324,56 @@ class PartialEvaluator {
         id: optionalContent.objId,
       };
     } else if (optionalContentType === "OCMD") {
+      const expression = optionalContent.get("VE");
+      if (Array.isArray(expression)) {
+        const MAX_NESTING = 10;
+
+        const parseExpression = (array, nestingCounter, currentResult) => {
```

Comment:
Can we please move this helper function to the top-level of the file, rather than defining it inline?